bitnami/mlflow/values.yaml

```diff
@@ -1421,3 +1422,20 @@ externalS3:
   protocol: "https"
   bucket: "mlflow"
   serveArtifacts: true
+
+## @section External Google Cloud Storage parameters
+## All of these values are only used when minio.enabled is set to false and externalS3 is not configured (host is empty)
+## @param externalGCS.bucket GCS bucket name. Activate gcs artifact storage if set
+## @param externalGCS.googleCloudProject Google Cloud Project to use (optional, needed when using "default application credentials")
+## @param externalGCS.useCredentialsInSecret Whether to read the GCS application credentials from a secret
+## @param externalGCS.existingSecret Name of an existing secret key containing the application credentials file (used only if useCredentialsInSecret is true)
+## @param externalGCS.existingSecretKey Key in the existing secret containing the application credentials (must be set if existingSecret is set)
```

Comment:
How does auth work if a user does not provide the credentials in a secret? Are these parameters actually optional?

Reply:
Oh that's nice. Thank you for providing the context!